### PR TITLE
[GSoD] Hardware Troubleshooting

### PR DESCRIPTION
**Pull request by PROTechThor**

### CODE REVIEW

File: user/troubleshooting/installation-troubleshooting.md [hunk 1]
```diff
@@ -77,6 +77,15 @@ If installing the available drivers does not help, disable the network card in t
...
 
 
+## "Unsupported Hardware Detected" error ##
+
+During Qubes installation, you may come across the error message which reads "Unsupported Hardware Detected. 
+Missing features: IOMMU/VT-d/AMD-Vi, Interrupt Remapping. Without these features, Qubes OS will not function normally". 
+
+This error message indicates that IOMMU-virtualization hasn’t been activated in the BIOS. 
+Return to the [hardware requirements][hardware-reqs] section to learn how to activate it. 
+If the setting is not configured correctly, it means that your hardware won’t be able to leverage some Qubes security features, such as a strict isolation of the networking and USB hardware.
```
Comment (marmarek) [inline user/troubleshooting/installation-troubleshooting.md:89]: This is getting more strict with recent versions:
- Qubes 4.0: default installation won't function properly without IOMMU, as default sys-net and sys-usb qubes require IOMMU; it's possible to configure then to reduce isolation and not use IOMMU by changing virtualization mode of those two VMs to "PV"
- Qubes 4.1: IOMMU is strictly required, even for PV; it is not possible to use Qubes on a system without IOMMU

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by PROTechThor** [comment]
@marmarek All good now?

**Review by marmarek — APPROVED** [review]

**Comment by marmarek** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

**Comment by fepitre** [comment]
PipelineRetry

**Comment by marmarek** [comment]
PipelineRetry

**Comment by andrewdavidwong** [comment]
I believe I'm just waiting for a signed approved from @marmarek before merging this.

**Comment by PROTechThor** [comment]
@marmarek Hi. Can you please give this a final review/approval so it can be merged already? It's been months already :)

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 1b9b4955740cf1bc6cf738dc2d28ae1080e54101
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmAl5lkACgkQ8yiUvpaE
k4qz2A/+JUYM8BBXrM+VFX553QWSDPb1ltOZNmGRVQxlSUxMAuLSB1eNOMTb8Hxl
DjuPAcCrmCI3z43yiB3x5h2bwq4tLRbWxn5ujiCtU8HWPsMzuCaBjUxAoi0t+ctS
8pG9yEhiFXLo5taEKdMNe2bux2NXDeQ5YvIFceTdIUDHoMFTdtUdUBXJkz/TFqZ4
yjnVR5cQhRCgFjwI+hdElXf4qebJ8mqlncf8aRZ8+BZ5KHNpPp/oMLTENU25ewMl
eo0ZO5n5/0KJwOan4qJkChV4oo/qvijoa72ta/gEPXIfCz6UTts734EEkYEXJS27
xcq9vEze9Eni9lOWAZBPPy7uA0tFbJFoIrC+J92f+dq2ZVol52r9UT9y0XInAzHx
rqce7SMa29XJA6a8o7l3NGpYbeFxZ2oFgCbVqm0R/2AXG4DCEpUhSY6+CzaHahhi
EDU8uo1SySzDzntzkdEFxQ5Y0+pTFWmQbXoX26GEH2QVr329AHEl+oOMtY8NISYv
OsXPU52p8ztVrW5AUqG0nhoRepY/jQWtHPt/aos3jk+9vGyuYRXnWqIyjtRRjQKo
FtlWYfZaXaBvlZS9+nT6WgYe7cU8X6awyPBVIoX/DbVMY31U5m7ekYH9tAYYZRdR
UkujQ7usPG7liCZThZY64ObdFXRllqO1RczYOObSZ4g/fkJLXX4=
=B/83
-----END PGP SIGNATURE-----
```

**Pull request merged**